src/services/reviews.js

```diff
@@ -120,11 +121,14 @@ module.exports = class reviewsHelper {
 				await handleComments(bodyData.comment, resourceId, userId)
 			}
 
+			// update the status if review assigned then update status as STARTED else CHANGES_REQUESTED
+			let status = common.REVIEW_STATUS_REQUESTED_FOR_CHANGES
+			if (startReview) {
+				status = common.REVIEW_STATUS_STARTED
+			}
```

Comment:
Can we do this with ternary operator?  @priyanka-TL 

const status = startReview ? common.REVIEW_STATUS_STARTED : common.REVIEW_STATUS_REQUESTED_FOR_CHANGES

Reply:
Also consider using const instead of let if its value is not updated elsewhere.

Reply:
consider using const if the value is not updated elsewhere.